reports/vamshi99.md

```diff
@@ -0,0 +1,209 @@
+   - link: https://gitlab.com/gitmate/open-source/IGitt/commit/4a044453f5d34c9f1b8227b7f57b8a0fd0caba57
+   - details: >
+      MergeRequest.py: Add fix_issues, close_issues, resolve_issues properties
+ - 7:
+   - repo: gitmate-2
+   - link: https://gitlab.com/gitmate/open-source/gitmate-2/commit/ffd554de617d619809f11b0284eaa032c2d9df83
+   - details: >
+      Auto assign issue based on commit mention in PR
+ - 8:
+   - repo: gitmate-2
+   - link: https://gitlab.com/gitmate/open-source/gitmate-2/commit/0eec2ec32e7afd7d017cad4c67a741c688e1bfc4
+   - details: >
+      apps.py: Add helper function to fetch plugin settings
+ - 9:
+   - repo: IGitt
+   - link: https://gitlab.com/gitmate/open-source/IGitt/commit/e2224d7d1f61a34065f741535e53e676dc51ff4b
+   - details: >
+      Add Team class
+ - 10:
+   - repo: gitmate-2
+   - link: https://gitlab.com/gitmate/open-source/gitmate-2/commit/f9d08c9a88c0b958a4c2ec5eb813fcd124b5300e
+   - details: >
+      Mark PR as WIP if coala plugin detects errors
+ - 11:
+   - repo: IGitt
+   - link: https://gitlab.com/gitmate/open-source/IGitt/commit/c4abdd4f9367af8ce73f39f525383def68de6b62
+   - details: >
+      MergeRequest: Add mergeable property
+ - 12:
+   - repo: gitmate-2
+   - link: https://gitlab.com/gitmate/open-source/gitmate-2/commit/db76142d3ecb1e6cd9cd10885ae2475cb07c4d3d
+   - details: >
+      Notify merge conflicts in a PR
+ - 13:
+   - repo: gitmate-2
+   - link: https://gitlab.com/gitmate/open-source/gitmate-2/commit/ad5480c9b0a124d98020a4ad3fd3caf20fa1e321
+   - details: >
+      pr_rejector: Support regex in branch names
+ - 14:
+   - repo: IGitt
+   - link: https://gitlab.com/gitmate/open-source/IGitt/commit/94735ff0f1024d6f293db90516087d8e4d2c353d
+   - details: >
+      Organization: Add issues property and filter_issues method
+ - 15:
+   - repo: IGitt
+   - link: https://gitlab.com/gitmate/open-source/IGitt/commit/fff0d568f1f2731c5055995ff19bbc93be842d5f
+   - details: >
+      Repository: Filter issues by label, assignee
+ - 16:
+   - repo: IGitt
+   - link: https://gitlab.com/gitmate/open-source/IGitt/commit/c025311167b07da5e825433a4d4d5e376adfed62
+   - details: >
+      GitLabUser: Fetch user id from username
+ - 17:
+   - repo: mr-rebaser
+   - link: https://gitlab.com/gitmate/open-source/mr-rebaser/commit/fad8305823a1058e93cd2397b5239a882b66c3f9
+   - details: >
+      Add squash pull request command
+ - 18:
+   - repo: mr-rebaser
+   - link: https://gitlab.com/gitmate/open-source/mr-rebaser/commit/3124c07ca5cd46acf985dc25fe22889e5a4744d0
+   - details: >
+      Dockerfile: Install requirements
+ - 19:
+   - repo: mr-rebaser
+   - link: https://gitlab.com/gitmate/open-source/mr-rebaser/commit/68360f3f0fac686907c7eb44c1a70bfa9e276d42
+   - details: >
+      run.py: Update squash command
+ - 20:
+   - repo: gitmate-2
+   - link: https://gitlab.com/gitmate/open-source/gitmate-2/commit/dda98a8ab67d01376d3199bb82260838cebab7b0
+   - details: >
+      rebaser: Add squash command
+ - 21:
```

Comment:
there are only 20 patches in your tarball.  needs to be the same as here, or you can include extra ones in your tarball if there are tiny/automated patches you dont want to include in this list.